fsutils/afero.go

```diff
@@ -0,0 +1,42 @@
+package fsutils
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/spf13/afero"
+)
+
+func SetupTemporaryFiles(fs afero.Fs) (file afero.File, dir string, err error) {
```

Comment:
I think a utils lib like this is useful, but I feel this function is too specific to get any reuse outside of github archive downloads. The resources it creates `tar-dir-` and `tar-file-` are pretty specific.
How about moving this to the helm chart package where it's used or renaming it something like `SetupTemporaryFilesForGitArchive`

Reply:
So I actually copied them over from solobot and didn’t want to modify or make private here so I could delete them there. Don’t mind leaving them in helm / install utils, but in the solobot case it’s for changelog verification, nothing to do with helm.

Reply:
On second thought, you're right -- added a better utility and moved these to private helpers. Should be pretty easy to make use of the new function in solobot.